projects/packages/jetpack-mu-wpcom/src/features/wpcom-sidebar-notice/wpcom-sidebar-notice.php

```diff
@@ -90,7 +90,7 @@ function wpcom_get_sidebar_notice() {
 	}
 
 	// Serialize message as object (on Simple sites we have an array, on Atomic sites we have an object).
-	$message = json_decode( wp_json_encode( $message[0] ) );
+	$message = json_decode( wp_json_encode( $message[0], JSON_UNESCAPED_SLASHES ) );
```

Comment:
Similar to the one above, may as well add `JSON_UNESCAPED_UNICODE` here.

```suggestion
	$message = json_decode( wp_json_encode( $message[0], JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ) );
```

Reply:
24d018fdee